_data/simple-icons.json

```
@@ -15,6 +15,11 @@
             "hex": "512BD4",
             "source": "https://github.com/dotnet/brand/tree/defe0408e765b48223a434a0d9a94213edc062f8"
         },
+        {
+            "title": "(ISC)²",
+            "hex": "007054",
+            "source": "https://www.isc2.org"
+        },
```

```suggestion
        {
            "title": "(ISC)²",
            "slug": "iscsquare",
            "hex": "007054",
            "source": "https://www.isc2.org"
        },
```

Their name is pronounced `ISC Square`.

You can find the reference in this video: https://youtu.be/VnwTyIb8OgY?t=27.

The SVG file name needs to be updated to same as the slug name.

---

I think in the video they say 'squared' - which would follow, given the mathmatical symbol used.

---

@LitoMore and @service-paradis - should we add `²` to our slug generation, so that the mathmatical symbol resolves correctly to a slug in future? Could also do `cubed` etc.

---

Good question @adamrusted! Since we only have a single icon having this, I think we might manage it by setting the `slug` property manually. If we encounter other icons with similar symbols, we might consider editing our slug generation algorithm.

On another note, should we add "ISC Squared" as an `aka`?

---

There are lots of well-known products named ISC. I prefer to reserve the `isc` slug for https://www.isc.org. Use `iscsquared` is more suitable.

For `ISC Squared`, it's not an official written form, and the `isc` pattern is easy enough to find the icon.